tests/integration/helm/helm-internal-auth_test.go

```diff
@@ -59,26 +59,28 @@ func (s *internalAuthHelmTest) TestHelmInstall() {
 		{
 			"builtinPlugins",
 			map[string]string{
-				"casSettings.env.FIFTYONE_AUTH_MODE": "internal",
+				"secret.fiftyone.fiftyoneDatabaseName": "fiftyone-int-bp-" + suffix,
+				"casSettings.env.FIFTYONE_AUTH_MODE":   "internal",
+				"casSettings.env.CAS_DATABASE_NAME":    "cas-int-bp-" + suffix,
 			},
 			[]serviceValidations{
 				{
 					name:             "teams-api",
-					url:              "https://local.fiftyone.ai/health",
+					url:              "", // "https://local.fiftyone.ai/health",
```

Comment:
Why are we commenting this and all the other `urls`?  This makes the integration tests not validate the endpoints?

Reply:
When you and I spoke about these, we decided to omit them because we didn't want the first iteration to wait for SSL/TLS certificates to be provisioned. With this setup, at 20 mins/cert, that would take these tests an incredibly long time to run

Reply:
Added a ternary operator so, in deployed envs, it skips checks. On minikube, it won't (as certs are already provisioned)

Reply:
Oh yes. This.  Maybe we can comment the code with a "why" they are being commented.  : )

Reply:
Comments added!